java/shared/pom.xml

```diff
@@ -59,8 +59,8 @@
     <dependency>
       <groupId>org.bitcoinj</groupId>
       <artifactId>bitcoinj-core</artifactId>
-      <version>0.16</version>
-      <!--Excluding protobuf-lite(newly added in 0.16 bitcoinj-core) as subzero uses protobuf-java-->
+      <version>0.16.1</version>
+      <!--Excluding protobuf-lite(newly added in 0.16.0 bitcoinj-core) as subzero uses protobuf-java-->
```

Comment:
great that it is honoring comments.

Reply:
This is a manual correction. Dependabot is almost smart enough to figure it out. 🙂